test/tests/redfish10/test_redfish10_api_taskservice.py

```diff
@@ -136,8 +132,7 @@ def test_redfish_v1_taskservice_tasklist(self):
         for member in members:
             tasklist.append(member['Id'])
         if fit_common.VERBOSITY >= 2:
-           print ("Task Service contains {0} tasks.".format(len(tasklist)))
-
+            print ("Task Service contains {0} tasks.".format(len(tasklist)))
```

Comment:
If you are using the python3 print() style, you will need to add the following import to the top of the file:
from __future__ import print_function

Otherwise, this will pass flake8, but print out 
```('Task Service contains 2 tasks.', )``` 
Alternative would be to convert to logs.stuff.